traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/fragments/CatalogItemDetailsFragment.java

```diff
@@ -140,6 +140,32 @@ public void onClick(View v) {
             }
         });
 
+        wishListToggleImageButton = view.findViewById(R.id.imagebutton_wishlist_toggle);
+        wishListToggleImageButton.setSelected(false);
+        wishListToggleImageButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (wishListToggleImageButton.isSelected()) {
+                    Toast.makeText(getContext(), "Removing from wishlist is not implemented yet!", Toast.LENGTH_SHORT).show();
+                } else {
+                    Traveler.addWishlist(catalogItemDetails.getId(), new WishlistAddCallback() {
+                        @Override
+                        public void onWishlistToggleSuccess() {
+                            Toast.makeText(getContext(), "Add wishlist success", Toast.LENGTH_SHORT).show();
+                        }
+
+                        @Override
+                        public void onWishlistToggleError(Error error) {
+                            Toast.makeText(getContext(), "Add wishlist error:" + error.getMessage(), Toast.LENGTH_SHORT).show();
```

Comment:
Follow the onError pattern in other fragments and activities...this is a UI deviation